src/app/modules/verification/verification.service.ts

```diff
@@ -257,17 +257,15 @@ const sendOTPForField = async (
   field: IVerificationFieldSchema,
   recipient: string,
   otp: string,
-): Promise<void> => {
+): Promise<unknown> => {
```

Comment:
why is this `unknown`? Looking through your code, seems like the return values are either `true` or error?

Reply:
I don't actually know what `MailService#sendVerificationOtp` returns, so better be safe than sorry and type strictly until we need to actually unwrap the values -- or should I also update MailService to return true on success? On second thoughts, it's probably good to standardize return values; will make mailservice return true too